src/modules/bulk-minter/components/Loading.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from "react";
+
+interface LoadingProps {
+    className?: string
+    w?: string
+    h?: string
+  }
```

Comment:
Not a big fan of trying to implement tailwind sizing with params like this (partially because really the `w-<size>` is an enum). Better to just let the caller define the size via `className`

```suggestion
interface LoadingProps {
    className?: string;
}
```